### ISSUE TITLE
Bug: Delete/Deletion of multiselected files deletes only 1st item

### ISSUE THREAD

**Issue by lopiuh** [eikek/docspell#919]
If I multiselect Items (more than one) and press delete only one item gets deleted and not the selected items.

Maybe a follow up because solr is not reachable and the loop stops after refreshing solr after the deletion of the 1st item is not possible? 

`docspell-restserver | [ioapp-compute-3] INFO  d.s.q.QAttachment - Have 1 attachments to delete. Must first delete archive entries 
docspell-restserver | [ioapp-compute-0] DEBUG d.s.q.QAttachment - Deleted 0 archive entries 
docspell-restserver | [ioapp-compute-0] DEBUG d.s.q.QAttachment - Deleting attachment: 5FnwG3F4qo5-GnPvDzFuV8X-Zckiyvs8f6d-vZxDU9rzSc2 
docspell-restserver | [ioapp-compute-0] DEBUG d.s.q.QAttachment - Deleted 4 meta records (source, meta, preview, archive). Deleting binaries now. 
docspell-restserver | [ioapp-compute-1] DEBUG d.b.ops.OItem - Remove item '6bWs9SwQKMg-53nyrCfy11w-pcVSxKmrENW-QeNKJRKvuf8' from index 
docspell-restserver | [docspell-restserver-blocking-5] INFO  o.h.s.m.Logger - HTTP/1.1 POST /api/v1/sec/items/deleteAll 
docspell-restserver | [docspell-restserver-blocking-5] INFO  o.h.s.m.Logger - service raised an error: class org.http4s.client.UnexpectedStatus 
docspell-restserver | [docspell-restserver-blocking-5] ERROR o.h.s.service-errors - Error servicing request: POST /api/v1/sec/items/deleteAll from 172.18.0.1 
docspell-restserver | org.http4s.client.UnexpectedStatus: unexpected HTTP status: 404 Not Found
docspell-solr | 2021-07-07 20:07:23.089 INFO  (qtp428696898-16) [   x:docspell] o.a.s.c.S.Request [docspell]  webapp=/solr path=/admin/ping params={} hits=0 status=0 QTime=140
docspell-solr | 2021-07-07 20:07:23.090 INFO  (qtp428696898-16) [   x:docspell] o.a.s.c.S.Request [docspell]  webapp=/solr path=/admin/ping params={} status=0 QTime=141
docspell-restserver | [blaze-selector-1] INFO  o.h.b.c.n.NIO1SocketServerGroup - Accepted connection from /127.0.0.1:53374 
docspell-restserver | [ioapp-compute-3] INFO  o.h.s.m.Logger - HTTP/1.1 GET / 
docspell-restserver | [ioapp-compute-3] INFO  o.h.s.m.Logger - HTTP/1.1 303 See Other 
docspell-restserver | [blaze-selector-2] INFO  o.h.b.c.n.NIO1SocketServerGroup - Accepted connection from /127.0.0.1:53376 
docspell-restserver | [ioapp-compute-0] INFO  o.h.s.m.Logger - HTTP/1.1 GET /app 
docspell-restserver | [ioapp-compute-0] INFO  o.h.s.m.Logger - HTTP/1.1 200 OK `

Greetings lopiuh

**Comment by eikek** [comment]
Yes, this is indeed a consequence of the other issue. It currently stops on first error, but actually I could just ignore solr errors here.

**Comment by lopiuh** [comment]
without solr errors multidelete works, shall I close this issue?

**Comment by eikek** [comment]
It's ok to leave it for now. I think I want to skip the solr errors in this case.
